### PR TITLE
Avionics Anonymous UAVCAN Peripherals

### PR DESCRIPTION
**Pull request by avionicsanonymous**
Adds pages for the Avionics Anonymous line of UAVCAN peripherals, and adds them to the positioning, rangefinder, and UAVCAN peripheral lists.

### CODE REVIEW

File: common/source/docs/common-avanon-laserint.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+`several common laser rangefinders <https://docs.avionicsanonymous.com/devices/laser_interface#currently-supported-lasers>`__ 
+that allows connection to Pixhawk and other similar autopilots via UAVCAN - A nice robust interface, no sketchy 
+long-distance I2C wiring and no hogged serial ports! 
+
+.. figure:: ../../../images/AvAnon-LaserInt.png
+   :target: ../_images/AvAnon-LaserInt.png
+
+   Avionics Anonymous Laser Interface
+
+The following parameter should be set on the autopilot (and then reboot the autopilot):
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-laserint.rst:17]: parameters

File: common/source/docs/common-avanon-gnss-mag.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+multi-constellation GNSS unit and a high-quality Memsic MMC5983MA magnetometer enabling sub-1-degree 
+heading accuracy. All of this interfaces to your autopilot via robust `UAVCAN <https://uavcan.org>`__ 
+interface - no sketchy I2C wiring for your magnetometer! 
+
+.. figure:: ../../../images/AvAnon-GNSSMag.png
+   :target: ../_images/AvAnon-GNSSMag.png
+
+   Avionics Anonymous GNSS/Mag: GPS and Compass
+
+The following parameter should be set on the autopilot (and then reboot the autopilot):
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-gnss-mag.rst:18]: parameters

File: common/source/docs/common-avanon-mag.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.. _common-avanon-mag:
+
+=========================================
+Avionics Anonymous Magnetometer - Compass
+=========================================
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-mag.rst:5]: Mgnetometer or Compass    choose one, the other is redundant...

File: common/source/docs/common-avanon-mag.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+The `Avionics Anonymous Magnetometer <https://docs.avionicsanonymous.com/devices/minimag>`__ is a high-performance
+magnetometer for small unmanned vehicles. Includes a high-quality Memsic MMC5983MA magnetometer enabling
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-mag.rst:8]: It includes...

File: common/source/docs/common-avanon-mag.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.. _common-avanon-mag:
+
+=========================================
+Avionics Anonymous Magnetometer - Compass
+=========================================
+
+The `Avionics Anonymous Magnetometer <https://docs.avionicsanonymous.com/devices/minimag>`__ is a high-performance
+magnetometer for small unmanned vehicles. Includes a high-quality Memsic MMC5983MA magnetometer enabling 
+sub-1-degree heading accuracy. All of this interfaces to your autopilot via robust `UAVCAN <https://uavcan.org>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-mag.rst:9]: All of these interface

File: common/source/docs/common-positioning-landing-page.rst [hunk 6]
```diff
@@ -14,6 +14,8 @@ positioning technologies:
 .. toctree::
     :maxdepth: 1
     
+    Avionics Anonymous GNSS/Mag: GNSS + Compass <common-avanon-gnss-mag>
```
Comment (Hwurzburg) [inline common/source/docs/common-positioning-landing-page.rst:17]: Drop GNSS/Mag     GNSS + Compass is sufficient

File: common/source/docs/common-positioning-landing-page.rst [hunk 7]
```diff
@@ -14,6 +14,8 @@ positioning technologies:
...
+    Avionics Anonymous Magnetometer: Compass <common-avanon-mag>
```
Comment (Hwurzburg) [inline common/source/docs/common-positioning-landing-page.rst:18]: Same here: Compass or Magnetometer, not both

File: common/source/docs/common-rangefinder-landingpage.rst [hunk 8]
```diff
@@ -53,6 +53,7 @@ based upon your set-up.
 
     Aerotenna US-D1 Radar Altimeter <common-aerotenna-usd1>
     Attollo Engineering Wasp200 <common-wasp200-lidar>
+    Avionics Anonymous Laser Interface <common-avanon-laserint>
```
Comment (Hwurzburg) [inline common/source/docs/common-rangefinder-landingpage.rst:56]: Name change as above please

File: common/source/docs/common-uavcan-peripherals.rst [hunk 9]
```diff
@@ -4,7 +4,10 @@
 UAVCAN Peripherals
 ==================
 
-
+- `Avionics Anonymous Air Data Computer <https://docs.avionicsanonymous.com/devices/microadc>`__
+- `Avionics Anonymous GNSS/Mag <https://docs.avionicsanonymous.com/devices/gnssmag>`__
+- `Avionics Anonymous Laser Interface <https://docs.avionicsanonymous.com/devices/laser_interface>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-uavcan-peripherals.rst:9]: name change please as requested above

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
the -adc page is not in any TOC, but dont worry about it...after you fix these, I will rebase, squash , and redo the UAVCAN peripherals page to a TOC format and add it there....and push to your PR, then merge....thanks

**Comment by avionicsanonymous** [comment]
@Hwurzburg , thanks for the thorough review! I believe I've addressed all of your comments.

Sorry for for harassing I2C if it felt like too much "marketing" - a bit too much copy/paste from my other pages.

**Comment by avionicsanonymous** [comment]
> 
> 
> the -adc page is not in any TOC, but dont worry about it...after you fix these, I will rebase, squash , and redo the UAVCAN peripherals page to a TOC format and add it there....and push to your PR, then merge....thanks

Thanks! I added the page but couldn't find any section at all for airspeed sensors, so figured I'd leave it orphaned until such a page was added.

File: common/source/docs/common-avanon-adc.rst [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+===============================================================================
+Avionics Anonymous Air Data Computer - Airspeed, Barometer, and Air Temperature
+===============================================================================
+
+The `Avionics Anonymous Air Data Computer <https://docs.avionicsanonymous.com/devices/microadc>`__ is tiny, 
+richly-featured Air Data Computer (ADC) for small unmanned airplanes. Includes an MS5611 barometric 
+pressure sensor for 10cm-resolution altitude, a Honeywell RSC-series differential pressure transducer 
+for extremely accurate airspeed, and a thermistor interface with high-resolution convertor for air 
+temperature measurement with less than 1 degree precision. All of this interfaces to your autopilot
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-avanon-adc.rst]: first change not doen

File: common/source/docs/common-avanon-gnss-mag.rst [hunk 11]
```diff
@@ -1,21 +1,21 @@
...
+===========================
 
 The `Avionics Anonymous GNSS/Mag <https://docs.avionicsanonymous.com/devices/gnssmag>`__ is a high-performance
-combination GPS and magnetometer for small unmanned vehicles. Includes a uBlox SAM-M8Q integrated 
-multi-constellation GNSS unit and a high-quality Memsic MMC5983MA magnetometer enabling sub-1-degree 
-heading accuracy. All of this interfaces to your autopilot via robust `UAVCAN <https://uavcan.org>`__ 
-interface - no sketchy I2C wiring for your magnetometer! 
+combination GPS and magnetometer for small unmanned vehicles. It includes a uBlox SAM-M8Q integrated 
+multi-constellation GNSS unit and a high-quality Memsic MMC5983MA magnetometer enabling better than 1-degree 
+heading accuracy. The devices interfaces to your autopilot via a robust `UAVCAN <https://uavcan.org>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-gnss-mag.rst:10]: These devices interface to ... 

or This device interfaces to...

File: common/source/docs/common-avanon-mag.rst [hunk 12]
```diff
@@ -1,18 +1,18 @@
...
+===============================
+Avionics Anonymous Magnetometer
+===============================
 
 The `Avionics Anonymous Magnetometer <https://docs.avionicsanonymous.com/devices/minimag>`__ is a high-performance
-magnetometer for small unmanned vehicles. Includes a high-quality Memsic MMC5983MA magnetometer enabling 
-sub-1-degree heading accuracy. All of this interfaces to your autopilot via robust `UAVCAN <https://uavcan.org>`__ 
-interface - no sketchy I2C wiring for your magnetometer! 
+magnetometer for small unmanned vehicles. It includes a high-quality Memsic MMC5983MA magnetometer enabling 
+better than 1-degree heading accuracy. The devices interfaces to your autopilot via a robust `UAVCAN <https://uavcan.org>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-avanon-mag.rst:9]: This device interfaces to...

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
I fixed the still outstanding change I requested, added a note on the rangefinder interface that I think would otherwise cause support calls to one of us, squash, rebased, changed the UAVCAN peripherals page to allow inclusion of the Airspeed sensor in a TOC, and merged....thanks

**Pull request merged**